vaccine-standby/README.md

```diff
@@ -0,0 +1,77 @@
+# COVID-19 Vaccine Standby List
+
+This Quick Deploy app is designed for public health agencies to create a COVID-19 vaccine eligibility standby list for their city or state. Residents send an SMS to a pre-configured Twilio phone number, and are asked a series of demographic questions informed by the CDC's vaccine rollout guidelines powered by a Twilio Studio Flow. Resident responses to the SMS chatbot are captured Studio Flow logs and displayed on a password protected section of the `index.html` page of this app.
```

Comment:
Can you remove Quick Deploy from this text and replace it with just `app` or `template`? since it can also be used through the CLI and in the future maybe from other places.

Reply:
Done